MediaBrowser.Model/Dlna/PlayToDeviceInfo.cs

```diff
@@ -0,0 +1,42 @@
+#pragma warning disable CS1591
```

Comment:
DeviceInfo.cs file with ToDeviceIndentification() removed as it's no longer required.